### PR TITLE
1st cpu donation

### PR DESCRIPTION
**Pull request by filip-strelec**
For the first donation of CPU collaboration, it would be wise to double-check if everything is ok since this is our first donation.

### CODE REVIEW

**Comment by kiriakos** [comment]
@filip-strelec unfortunately the yml is cant be consumed by the CI in this state. The format requires an `ada` and an equivalent `usd` value in the donation body. Other than that everything looks good. If I may nit pick, a cardano TX next to the litecoin tx in `intermediate_txs` would aid transparency but that's not too critical.

**Comment by filip-strelec** [comment]
> @filip-strelec unfortunately the yml is cant be consumed by the CI in this state. The format requires an `ada` and an equivalent `usd` value in the donation body. Other than that everything looks good. If I may nit pick, a cardano TX next to the litecoin tx in `intermediate_txs` would aid transparency but that's not too critical.

Hey,  I made the necessary changes for ada/usd values, regarding intermediate_txs, if I realised correctly, it would be good to add a prior Wallet->binance transaction? where I exchanged ada for ltc?

Also I'm sending ada from CPU wallet to daedalus wallet, every couple of months, where it's easier to manage that ada within daedalus GUI... should i also add that transaction for intermediate_txs?



Thank you @kiriakos

**Review by way2many — APPROVED** [review]
Seems good for me. The Litecoin trail might be hard to follow if I understand you reply. It is ADA rewards -> Daedalus wallet -> LTC exchanged on Binance -> Donation in LTC.

I'll approve and merge. If needed a new PR can be made to have the ADA -> LTC trail.

**Comment by filip-strelec** [comment]
Great, thank you @way2many I will see for the next donation what to do regarding intermediate_txs

yes, you understood correctly btw. :)
